SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -178,6 +178,18 @@ void USpatialReceiver::FlushRemoveComponentOps()
 	QueuedRemoveComponentOps.Empty();
 }
 
+void USpatialReceiver::RemoveComponentOpsForEntity(Worker_EntityId EntityId)
```

Comment:
Do we also queue up add component ops that arrive outside of critical sections? If so we need to remove them too.

Reply:
Yep, they get queued up in the same array.

Reply:
Where does that happen. It looks like this is an array of remove component ops specifically?

Reply:
Discussed offline - it's fine.